app/pastes/pasteRepository.js

```diff
@@ -15,6 +17,11 @@ class PasteRepository {
         }
 
         const result = await this.findOne({ id });
+        // added text here, not in findAll method, because findAll seems like a general search for pastes and
+        // we get text only when we open specific paste.
+        // Also here because i was thinking how to implement it in findAll and it was shit
+        const text = await this.textStorage.getText(result._id);
+        result._text = text;
 
```

Comment:
btw, this "_anything" is ugly. We learned how to deal with many times already :/